packages/bee-q/src/components/panel/scss/bq-panel-variables.scss

```
@@ -0,0 +1,37 @@
+/* -------------------------------------------------------------------------- */
+/*                          Panel custom properties           */
+/* -------------------------------------------------------------------------- */
+
+:host {
+  /**
+  * @prop --bq-panel--background: Panel background color
+
+  * @prop --bq-panel--border-color: Panel border color
+  * @prop --bq-panel--border-radius: Panel border radius
+  * @prop --bq-panel--border-style: Panel border style
+  * @prop --bq-panel--border-width: Panel border width
+
+  * @prop --bq-panel--box-shadow: Panel box shadow
+  * @prop --bq-panel--padding: Panel padding 
+
+  * @prop --bq-panel--height: Panel height
+  * @prop --bq-panel--width: Panel width
+  * @prop --bq-panel-z-index: Panel z-index applied when opened
+  */
+
+  --bq-panel--background: theme('colors.bg.primary');
+
+  --bq-panel--border-color: theme('colors.transparent');
+  --bq-panel--border-radius: theme('borderRadius.s');
+  --bq-panel--border-style: none;
+  --bq-panel--border-width: unset;
+
+  --bq-panel--box-shadow: theme('boxShadow.m');
+  --bq-panel--paddingY: theme('spacing.xs');
+  --bq-panel--padding-start: theme('spacing.xs');
+  --bq-panel--padding-end: theme('spacing.xs');
+
+  --bq-panel--height: 200px;
```

I would rather recommend setting the panel height relative to the view height, e.g:

```suggestion
  --bq-panel--height: 50vh;
```

---

Ok, but isn't `50vh` too much for a dropdown?